content/applications/general/iot/config/ssh_connect.rst

```diff
@@ -0,0 +1,47 @@
+==========================
+Connect an IoT box via SSH
+==========================
+
+.. |iot| replace:: :abbr:`IoT (Internet of Things)`
+.. |SSH| replace:: :abbr:`SSH (secure shell protocol)`
+
+To provide an |SSH| connection to an Internet of Things (IoT) box, a password needs to be generated.
+
+.. warning::
+   This feature should **only** be utilized with trusted parties, as it provides administrative
+   access to the |iot| box, which can create security issues.
+
+   Managing an |SSH| connection is **not** covered under the standard scope of Odoo support. Visit
```

Comment:
This could be an important admonition or a note  if you want

Reply:
I wanted to keep it all under the same admonition (I didn't want to add _too_ many blocks).